configure.ac

```diff
@@ -1622,6 +1630,10 @@ AC_MSG_CHECKING([whether to build bitcoin-util])
 AM_CONDITIONAL([BUILD_BITCOIN_UTIL], [test $build_bitcoin_util = "yes"])
 AC_MSG_RESULT($build_bitcoin_util)
 
+AC_MSG_CHECKING([whether to build experimental bitcoin-chainstate])
+AM_CONDITIONAL([BUILD_BITCOIN_CHAINSTATE], [test x$build_bitcoin_chainstate = xyes])
```

Comment:
nit: remove x and add missing "quotes" like everywhere else?

Reply:
Done as of 9050346f04d0435ad71d93ba1df4f5c126c0337b